cluster-autoscaler/core/scaledown/budgets/budgets_test.go

```diff
@@ -234,6 +259,24 @@ func TestCropNodesToBudgets(t *testing.T) {
 			wantEmpty: append(generateNodeGroupViewList(atomic3, 0, 3), generateNodeGroupViewList(testNg, 0, 5)...),
 			wantDrain: []*NodeGroupView{},
 		},
+		"empty&drain regular and atomic nodes in same group exceeding overall limit, no deletions in progress": {
+			empty:     append(generateNodeGroupViewList(testNg, 0, 5), generateNodeGroupViewList(atomic11, 0, 8)...),
+			drain:     generateNodeGroupViewList(atomic11, 8, 11),
+			wantEmpty: generateNodeGroupViewList(atomic11, 0, 8),
+			wantDrain: generateNodeGroupViewList(atomic11, 8, 11),
+		},
+		"empty&drain regular and multiple atomic nodes in same group exceeding drain limit, no deletions in progress": {
+			empty:     append(append(generateNodeGroupViewList(testNg, 0, 5), generateNodeGroupViewList(atomic8, 0, 5)...), generateNodeGroupViewList(atomic11, 0, 8)...),
+			drain:     append(generateNodeGroupViewList(atomic11, 8, 3), generateNodeGroupViewList(atomic8, 5, 8)...),
```

Comment:
`generateNodeGroupViewList(atomic11, 8, 3)` - shouldn't `from < to`? Now this returns no nodes

Reply:
Nice catch! It should be correct now.